roles/collect/templates/manifest.j2

```diff
@@ -1,5 +1,5 @@
 {
-    "file": "{{ collect_csv_filename }}",
+    "file": "[{{ csv_files|join(',') }}]",
```

Comment:
we were going to change this to `"files": [ ... ]`, right?

Reply:
We did. Refresh your view. It was pushed in 45579ad